wrappers/python/examples/align-depth2color.py

```diff
@@ -24,6 +25,14 @@
 pipeline_profile = config.resolve(pipeline_wrapper)
 device = pipeline_profile.get_device()
 device_product_line = str(device.get_info(rs.camera_info.product_line))
+deviceList = context.query_devices()
+dev = deviceList.front()
+sensorsList = dev.query_sensors()
+try:
+    next(s for s in sensorsList if s.get_info(rs.camera_info.name) == 'RGB Camera')
+except Exception as e:
+    print("The connected device does not support RGB stream")
+    exit(0)
```

Comment:
Replace `try/catch` with check routine in python similar to C++ examples. Here and in the next two demos

Reply:
I need more clarification here: 
1. There is no common file for python examples where I can put the check method and I don't think creating a common file especially for that is reasonable. What do you suggest ?
2. Should the method be exactly as in C++ (checking serial etc ) ?